packages/eslint-config-react/package.json

```diff
@@ -24,19 +24,16 @@
     "node": "^8 || ^10 || ^12"
   },
   "dependencies": {
-    "@zonedigital/eslint-config-zone": "^4.3.0"
-  },
-  "devDependencies": {
-    "eslint-plugin-compat": "^3.0.0",
-    "eslint-plugin-import": "^2.14.0",
+    "@zonedigital/eslint-config-zone": "^4.3.0",
     "eslint-plugin-react": "^7.13.0",
     "eslint-plugin-react-hooks": "^2.0.0"
   },
+  "devDependencies": {
+    "eslint": "^6.5.1",
+    "prettier": "^1.18.2"
+  },
   "peerDependencies": {
     "eslint": "^6.0.0",
-    "eslint-plugin-compat": "^3.0.0",
-    "eslint-plugin-import": "^2.0.0",
-    "eslint-plugin-react": "^7.13.0",
-    "eslint-plugin-react-hooks": "^2.0.0"
+    "prettier": "^1.0.0"
```

Comment:
any reason we wouldnt bump this up to be inline with the devDependencies verison? https://github.com/zone/frontend/pull/103/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12 ?

Reply:
Mainly to provide a wider version range for the host project.